src/Banking/Transaction.php

```diff
@@ -22,6 +22,7 @@ class Transaction implements \JsonSerializable
     private $valueTimestamp = 0;
     private $entryTimestamp = 0;
     private $transactionCode = '';
+    private $virtualAccount = '';
```

Comment:
just what is a virtualAccount? This doesn't seem to belong in the 'default' Transactions for other banks... ?

Reply:
ok, the virtual account has been stripped from the base class.